ModernRonin.FluentArgumentParser/Help/HelpMaker.cs

```diff
@@ -184,7 +186,14 @@ Row indexableToRow(AnIndexableParameter indexable)
         Row optionalToRow(OptionalParameter opt)
         {
             var result = indexableToRow(opt);
-            result.RightLines.Add($"default: {opt.Default}");
+            if (opt.Default == null)
+            {
+                result.RightLines.Add(opt.Description);
+            }
+            else
+            {
+                result.RightLines.Add($"Default: {opt.Default}");
+            }
```

Comment:
this can be simplified to `result.RightLines.Add(opt.Default is null ? opt.Description : $"Default: {opt.Default}"`